### PR TITLE
Add option for JVMTI watched field performance

### PR DESCRIPTION
**Pull request by SueChaplain**
New option for turning on experimental JIT performance improvements
to support JVMTI watched fields.

- new topic
- update to release topic
- entry in Table of Contents

Closes: #252

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
@gacholio @DanHeidinga - please review.

File: docs/xxjitinlinewatches.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:\[+|-\]JITInlineWatches
```
Comment (fjeremic) [inline docs/xxjitinlinewatches.md]: @gacholio I thought we were going to deprecate this once all platforms support it? I was under the impression that we will not be externalizing this option. Do we still plan to do this?

Comment (gacholio) [reply docs/xxjitinlinewatches.md]: That was my feeling as well, but others overruled.

Comment (DanHeidinga) [reply docs/xxjitinlinewatches.md]: We want to enable users to get the benefits in `0.14.0` and to make it the default in `0.15.0`.  Having the option documented makes it easier to enable / disable if any issues surface.

**Comment by SueChaplain** [comment]
@DanHeidinga  or @pshipton - please can you merge. No other committers in the docs team this week.

**Comment by DanHeidinga** [comment]
Merged.

**Pull request merged**